dbms/src/Storages/tests/gtest_filter_parser.cpp

```diff
@@ -367,6 +368,39 @@ try
         std::regex rx(R"(\{"op":"or","children":\[\{"op":"unsupported",.*\},\{"op":"not","children":\[\{"op":"equal","col":"col_2","value":"666"\}\]\}\]\})");
         EXPECT_TRUE(std::regex_search(rs_operator->toDebugString(), rx));
     }
+
+    {
+        // And with IsNULL
+        auto rs_operator = generateRsOperator(table_info_json, "select * from default.t_111 where col_2 = 789 and col_3 is null");
+        EXPECT_EQ(rs_operator->name(), "and");
+        EXPECT_EQ(rs_operator->getAttrs().size(), 2);
+        EXPECT_EQ(rs_operator->getAttrs()[0].col_name, "col_2");
+        EXPECT_EQ(rs_operator->getAttrs()[0].col_id, 2);
+        EXPECT_EQ(rs_operator->getAttrs()[1].col_name, "col_3");
+        EXPECT_EQ(rs_operator->getAttrs()[1].col_id, 3);
+        EXPECT_EQ(rs_operator->toDebugString(), "{\"op\":\"and\",\"children\":[{\"op\":\"equal\",\"col\":\"col_2\",\"value\":\"789\"},{\"op\":\"isnull\",\"col\":\"col_3\"}]}");
+    }
```

Comment:
do we support "col_3 is not null" parsed as RSOperator?

Reply:
yes, it will be parse like not(isnull(xxx))

Reply:
And TiDB will transfer `is not null` to `not(isnull(xx)`.